### PR TITLE
add CMake Flag for gcc 4.8 to disable AVX512-specific optimizations

### PR DESCRIPTION
**Pull request by TingDaoK**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: builder/core/data.py [hunk 1]
```diff
@@ -555,7 +555,10 @@ class PKG_TOOLS(Enum):
         'yum_compiler_packages': ['gcc', 'gcc-c++'],
 
         'versions': {
-            '4.8': {},
+            '4.8': {
+                'cmake_args': [
```
Comment (DmitriyMusatkin) [inline builder/core/data.py]: dont think, this is the right place. we have separate flags for aws-lc https://github.com/awslabs/aws-crt-builder/blob/a31bfacf48d2d10497397674c9928238531b7a50/builder/imports/awslc.py

Comment (TingDaoK) [reply builder/core/data.py]: but that should be a gcc4.8 only flag, right?

Comment (DmitriyMusatkin) [reply builder/core/data.py]: it is. this is aws-lc only flag so arguably we should not be passing it to everything that builds with 4.8. 
ideally we would specify specifically for aws-lc that it should build with that flag on 4.8, but lc does not support builder. file i linked above is where aws-lc specific config gets added, we should be able to add compiler specific cmake there

Comment (TingDaoK) [reply builder/core/data.py]: Yep, that's better, updated

**Review by DmitriyMusatkin — APPROVED** [review]

**Pull request merged**